frontend/tsconfig.json

```diff
@@ -7,6 +7,14 @@
     "importHelpers": true,
     "isolatedModules": true,
     "noEmit": true,
+    "paths": {
+      "@/*": [
+        "./src/*"
+      ],
+      "@/common": [
+        "../common/*"
+      ]
+    },
     "types": [
       "@intlify/vite-plugin-vue-i18n/client"
```

Comment:
This can now go as well 😭

Reply:
Hey, I don't tell you the news, TradingVueJS doesn't work either with Webpack, so we can keep `vite`

Reply:
wohooo ;)